src/components/Balance/index.js

```diff
@@ -4,8 +4,18 @@ import { StyledBalance, StyledIncomeExpenses, StyledIncome, StyledExpenses } fro
 
 const Balance = () => {
   const { transactions } = useTransactionContext();
-  const income = transactions.filter((item) => item.type === 'income').reduce((sum, item) => sum += item.amount, 0).toFixed(2);
-  const expenses = transactions.filter((item) => item.type === 'expense').reduce((sum, item) => sum += item.amount, 0).toFixed(2);
+  const income = transactions.reduce((sum, item) => {
```

Comment:
this is exactly what I meant, I wanted it to fit on one line though XD (but nvm.. this works)

`transactions.reduce((sum, item) => sum += (item.type === 'Income'? item.amount: 0), 0)`